gatsby-config.js

```diff
@@ -146,7 +152,9 @@ module.exports = {
         // enablePartialUpdates: true,
         queries: require("./src/utilities/algoliaQueries"),
         // we skip the indexing completely on non-prod builds.
-        skipIndexing: true,
+        skipIndexing:
+          !!process.env.ALGOLIA_SKIP_INDEXING || // Manually skip indexing
+          process.env.CONTEXT !== "production", // Never index for non-prod
```

Comment:
Perhaps reverse the logic on the SKIP functionality and opt-in on an environment so it defaults to not indexing. Does the `process.env.CONTEXT` check cover the scenario of using `netlify dev` while linked to the production website?

Reply:
```suggestion
		  process.env.NETLIFY &&
          process.env.ALGOLIA_RUN_INDEXING &&
          process.env.CONTEXT == "production",
```

How about something like this to prevent running index on any environment that isn't netlify prod with a specific env var to turn ON the index? I think this covers local, netlify dev, and the dev site.